.gitignore

```diff
@@ -82,6 +82,7 @@ docs/*/pdfreactor-fonts.html
 #################
 *.egg-info
 .installed.cfg
+static/
```

Comment:
@anselmbradford it will live in the cfgov folder & any app directories.
this will ignore all static folders in the project. since gulp will process files and dump them in there, not be committed